Gemfile.jruby-1.9.lock

```diff
@@ -167,16 +167,16 @@ GEM
       i18n (= 0.6.9)
       jrjackson (~> 0.3.7)
       jruby-openssl (= 0.9.13)
-      logstash-core-event (= 2.3.1.snapshot1)
+      logstash-core-event (= 2.3.1)
```

Comment:
Since I only publish the `logstash-core-event` using jruby the gems is only available on the jruby platform.